modules/Terminal-Basics/exercises/Environment-Variables-And-PATH.md

```diff
@@ -85,7 +85,6 @@ Research these links and topics to learn more about the `$PATH`. While you're re
 0. [ ] Add a new directory to your PATH
 0. [ ] Add a new directory to the beginning of your PATH
 0. [ ] Add a new directory to the end of your PATH
-0. [ ] Export an environment variable and access it in a new shell window
```

Comment:
looks good. I'd like for us to have an exercise for adding a PATH modification to your `~/.bashrc`. Do we have one somewhere other than here? If not we should add one